signalfx-java/src/main/java/com/signalfx/signalflow/ComputationAbortedException.java

```diff
@@ -0,0 +1,39 @@
+package com.signalfx.signalflow;
+
+import java.util.Map;
+
+/**
+ *  Exception thrown if the computation is aborted during its execution.
+ */
+public class ComputationAbortedException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    protected String state;
+    protected String reason;
+
+    public ComputationAbortedException(Map<String, Object> abortInfo) {
```

Comment:
k .. I wasn't 100% sure if it would always be string for value so sided on safety.  will make the change